Web/Sphere/rollup.config.js

```diff
@@ -1,13 +1,25 @@
 import typescript from "@rollup/plugin-typescript";
 
-export default args => {
-    const result = args.configDefaultConfig;
-    result.forEach(config => {
-        config.plugins.push(
-            typescript({
-                include: ["../../Shared/ComponentParent/**/*.ts+(|x)", "./**/*.ts+(|x)"]
-            })
-        );
-    });
-    return result;
+import commonjs from "@rollup/plugin-commonjs";
```

Comment:
Why just not create "shared" rollup config?

Reply:
Done!